### DISCUSSION TITLE
posts/berkenalan-dengan-scout

### DISCUSSION THREAD

**Discussion by giscus** [teknologi-umum/blog#148] [General]
# posts/berkenalan-dengan-scout

Membuat searching data makin mudah dan cepat (udah kayak tagline isp ga tuh)

https://teknologiumum.com/posts/berkenalan-dengan-scout

**Comment by wiratama13** [comment]
Mantab bang👍👍

**Reply by yuxxeun** [reply]
thanks

**Comment by AdiCahyaSaputra** [comment]
keren bre 🔥

**Reply by yuxxeun** [reply]
you too 🔥

**Comment by omjipanggg** [comment]
Mas, saya Aji dari Kota Serang, saya ada pertanyaan, saya mohon izin hubungi sampeyan via email ya (kalau boleh, semoga boleh), terima kasih banyakkk🙅‍♂️🙆‍♂️🙏

**Reply by yuxxeun** [reply]
Hallo, wah baru baca komentar, silahkan hubungi saya via email kridospace@gmail.com
